src/analysis/_testData/testData.ts

```diff
@@ -215,6 +216,18 @@ export const defaultTestDisk = {
   blockSize: 4096,
 };
 
+export const defaultGoogleBucketOptions = {
```

Comment:
```suggestion
export const defaultGoogleBucketOptions: SomeType = {
```

It would be nice to add types to these to these to ensure that the test data stays up to date with the actual interface.